docs/dev/plugins/cloudFoundry/CloudFoundry_Planning.md

```diff
@@ -0,0 +1,37 @@
+# Cloud Foundry(CF) - Planning
+This document show the planning and the next steps for the CF-plugin.
+
+## Playground
+To test some CF specific functionalities and to determine how the different node-types could be implemented it is useful to have a CF environment.
+Therefore we have an "test-account" on pivotal.
+For automatical testing it is probably useful to checkout how to implement CF on vSphere.
+
+### Tests
+To create end-to-end tests the java library [CF-JavaClient](https://github.com/cloudfoundry/cf-java-client) might be useful. Using it in the plugin-implementation it is not a option, because the plugin only creates the target artifact and does not connect to the cf instance. (But this decision is not finally. Has to be discussed!)
+
+## Next Steps
+
+### Open Issues
```

Comment:
It probably seems like a better idea to put this into issues instead of a list like that.

Maybe consider opening issues for each point listed here and adding them to a "Cloud foundry" epic.

Reply:
I want to keep them in this list but add it to the issue board as well.